src/app/Media/VoIPMediaSession.cs

```diff
@@ -301,6 +340,12 @@ protected void RtpMediaPacketReceived(IPEndPoint remoteEndPoint, SDPMediaTypesEn
 
                 Media.AudioSink.GotAudioRtp(remoteEndPoint, hdr.SyncSource, hdr.SequenceNumber, hdr.Timestamp, hdr.PayloadType, marker, rtpPacket.Payload);
             }
+            else if (mediaType == SDPMediaTypesEnum.text && Media.TextSink != null)
+            {
+                logger.LogTrace($"{nameof(RtpMediaPacketReceived)} text RTP packet received from {remoteEndPoint} ssrc {hdr.SyncSource} seqnum {hdr.SequenceNumber} timestamp {hdr.Timestamp} payload type {hdr.PayloadType}.");

```

Comment:
As per #1306, use structured semantic logging instead of dynamic log message templates.